python/ray/tests/test_raydp_dataset.py

```diff
@@ -0,0 +1,44 @@
+import pytest
```

Comment:
You need to move this file to `python/ray/data/tests` for the data bazel test rule to automatically pick it up.